crates/astria-sequencer/src/sequence/storage/values.rs

```diff
@@ -0,0 +1,56 @@
+use std::fmt::{
+    self,
+    Display,
+    Formatter,
+};
+
+use astria_eyre::eyre::bail;
+use borsh::{
+    BorshDeserialize,
+    BorshSerialize,
+};
+
+#[derive(Debug, BorshSerialize, BorshDeserialize)]
+pub(crate) enum Value {
+    Fee(Fee),
+}
+
+impl Display for Value {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        match self {
+            Value::Fee(fee) => write!(f, "fee {}", fee.0),
+        }
+    }
+}
+
+#[derive(Debug, BorshSerialize, BorshDeserialize)]
+pub(crate) struct Fee(u128);
+
+impl From<u128> for Fee {
+    fn from(fee: u128) -> Self {
+        Fee(fee)
+    }
+}
+
+impl From<Fee> for u128 {
+    fn from(fee: Fee) -> Self {
+        fee.0
+    }
+}
+
+impl<'a> From<Fee> for crate::storage::StoredValue<'a> {
```

Comment:
With the visibility changes we discussed offline, meaning these low-level types become private to the relevant component's module, I think it's not harmful to support these `From` impls as is.  I'm not sure I see any benefit to putting in the extra step, and it would be quite a lot more boilerplate.

Reply:
Yeah, in this case I am fine with it.